cfg/param.yaml

```diff
@@ -1,7 +1,7 @@
-depthimage_to_laserscan:
+depthimage_to_laserscan_node:
```

Comment:
The current configuration matches what is in the source code: https://github.com/ros-perception/depthimage_to_laserscan/blob/7b60f8d3396a8e5904f27ac5d910366627a42394/src/DepthImageToLaserScanROS.cpp#L49 .  Can you explain a bit more why you needed this change?

Reply:
That is correct. Sorry for the late reply. I tried to figure out whats the cause but unfortunately couldn't figure out why the "_node" is appended in the parameter namespace. When I run: `$ ros2 param list` on a fresh cloned ros2 branch, the output is:
```
/depthimage_to_laserscan_node:
  output_frame
  qos_overrides./parameter_events.publisher.depth
  qos_overrides./parameter_events.publisher.durability
  qos_overrides./parameter_events.publisher.history
  qos_overrides./parameter_events.publisher.reliability
  range_max
  range_min
  scan_height
  scan_time
  use_sim_time
/rqt_gui_py_node_7891:
  use_sim_time
```
So it alsways has "/depthimage_to_laserscan_**node**" as namespace in ros humble.

Reply:
Oh, I think I see what is happening.  I suspect that you are using the depthimage node from the launch file.

And there is indeed a bug that the launch file is renaming the node to `depthimage_to_laserscan_node`: https://github.com/ros-perception/depthimage_to_laserscan/blob/f7b24c3f10b04218abc42b3621c7838aca8cafa8/launch/depthimage_to_laserscan-launch.py#L31 .

I think what we should do here is to fix the launch file so that the name is `depthimage_to_laserscan`.  That should make it match the source code, and the other launch file.

Reply:
Great, thank you for your guidance! I modified as you suggested.